denonavr/denonavr.py

```diff
@@ -1441,8 +1443,10 @@ def match_sound_mode(self, sound_mode_raw):
             return sound_mode
         except KeyError:
             pass
-        _LOGGER.warning("Not able to match sound mode, "
-                        "returning raw sound mode.")
+        if sound_mode_raw not in self._error_list:
```

Comment:
Please add sound_mode_raw to self._sm_match_dict in case of a KeyError instead of creating the self._error_list attribute.

Reply:
with a mapping to itself